src/bin/pgcopydb/pgsql.c

```diff
@@ -3077,7 +3072,7 @@ pg_copy_send_query(PGSQL *pgsql, CopyArgs *args, ExecStatusType status)
 	}
 	else if (status == PGRES_COPY_IN)
 	{
-		if (args->dstAttrList != NULL && !streq(args->dstAttrList, ""))
+		if (!streq(args->dstAttrList, ""))
```

Comment:
Why can't that thing be NULL?

Reply:
The reason for the change is that `streq` already has the null checks by definition.